src/quagga/build.sh

```diff
@@ -2,34 +2,14 @@
 
 mkdir quagga
 
-# Get debian source for 0.9.24.1-2
-wget -O quagga_0.99.24.1.orig.tar.gz 'https://sonicstorage.blob.core.windows.net/packages/quagga_0.99.24.1.orig.tar.gz?sv=2015-04-05&sr=b&sig=7g3AC%2FkoX3wYztJYtXFt6Wl7zj%2BYwLkbXVNaSaRvUDU%3D&se=2026-07-21T00%3A07%3A31Z&sp=r'
-tar -xzf quagga_0.99.24.1.orig.tar.gz --strip-components=1 -C quagga
-ls -lrt
+wget -O quagga_0.99.24.1.tar.gz https://github.com/Azure/sonic-quagga/archive/debian/0.99.24.1.tar.gz
```

Comment:
How about create a git-submodule at /src/sonic-quagga, and remove this directory?